google_compute_engine/accounts/tests/accounts_daemon_test.py

```diff
@@ -57,8 +57,8 @@ def testAccountsDaemon(
           mock.call.watcher.MetadataWatcher(logger=mock_logger_instance),
           mock.call.utils.AccountsUtils(
               logger=mock_logger_instance, groups='foo,bar', remove=True,
-              useradd_cmd=mock.ANY, userdel_cmd=mock.ANY, usermod_cmd=mock.ANY,
-              groupadd_cmd=mock.ANY),
+              gpasswd_cmd=mock.ANY, useradd_cmd=mock.ANY, userdel_cmd=mock.ANY,
```

Comment:
Small nit: mind moving `groupadd` next to `gpasswd` so that it's mostly alphabetized (and consistent with the ordering in `accounts_utils`)?